src/views/LotteryV2/index.tsx

```diff
@@ -48,14 +42,15 @@ const LotteryV2 = () => {
 
   return (
     <LotteryPage>
-      <PageSection background={TITLE_BG} svgFill={theme.colors.overlay} index={4}>
+      <PageSection background={TITLE_BG} svgFill={theme.colors.overlay} index={3} hasCurvedDivider={false}>
         <Hero />
       </PageSection>
-      <TicketsSection
-        containerProps={{ style: { margin: '0' } }}
+      <PageSection
+        containerProps={{ style: { marginTop: '-30px' } }}
         background={GET_TICKETS_BG}
-        hasCurvedDivider={false}
-        index={3}
+        concaveDivider
+        curvePosition="top"
+        index={4}
       >
         <Flex alignItems="center" justifyContent="center" flexDirection="column">
```

Comment:
Let's add a bit of padding (top) to this? It gets v tight to the PageSection above, particularly on mobile.